init

```diff
@@ -4,6 +4,8 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 HOMEDIR=$(eval echo "~`whoami`")
 source /etc/environment || :
+# Turn off coredumps that seem to happen on smaller, overloaded instances leading to closed-loop feedback
+sudo /usr/bin/sed -i 's/#Storage=external/Storage=none/' /etc/systemd/coredump.conf
```

Comment:
I kind of want to say that we may want to move this logic into the CFN template userdata. But I don't know.